### DISCUSSION TITLE
Problem with connection to hidden WiFi

### DISCUSSION THREAD

**Discussion by pamidev** [lora-aprs/LoRa_APRS_iGate#152] [General]
Hello everybody.

I noticed that my Lora APRS iGate on Lora32 v2.1_1.6 board does not connect to the hidden WiFi (no broadcasting SSID). I do everything according to the instructions, but my LoRa board does not connect to my WiFi. I exclude incorrect WiFi SSID and password. Have anyone the same problem? Can anyone confirm this? 

73, pamidev

**Comment by OH2LAK** [comment]
Pawel, does the problem persist with the particular wifi network if you
make it visible?

73,
Erik OH2LAK

**Reply by pamidev** [reply]
I have checked and confirmed. My LoRa iGate works with visible WIFi SSID.

73, pamidev

**Reply by AmyYummy** [reply]
Is it showing that it is connected to wifi, but no network speed?

If this is the case, it is recommended that you log into the [router](www.baudcom.cn) and check if the router is connected to the Internet.

**Reply by pamidev** [reply]
No, no, no... My router works fine. The router is connected to the Internet. Conclussion: board TTGO Lora32 ver. 2.1_1.6 not works with hidden WiFi SSID (not connect to the Internet).

73, pamidev

**Comment by peterus** [comment]
please check your channel:

https://github.com/espressif/arduino-esp32/issues/3961

looks also like some arduino-core versions and hw-versions do not support hidden wifis.

**Reply by pamidev** [reply]
How can this be fixed? My router auto-selects the clearest channel.

73, pamidev
